Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCardTests.swift

```diff
@@ -6,7 +6,9 @@ import XCTest
 
 final class PPOProjectCardTests: TestCase {
   let size = CGSize(width: 375, height: 700)
-  func testAddressLocks() {
+
+  @MainActor
+  func testAddressLocks() async {
```

Comment:
Oh interesting. My understanding is that objects that use `@StateObject` or `@ObservableObject`, like the card and view model being used here, have the benefit of `@MainActor` essentially built in to them. 

Do the `async/await` calls force updates to the background thread if we don't explicitly add this attribute?

Reply:
Without this it led to a deadlock when the test was running.